src/qml/main.qml

```diff
@@ -48,6 +48,11 @@ Kirigami.ApplicationWindow {
 					// open login page
 					pageStack.push(loginPage);
 				}
+			},
+			Kirigami.Action {
+				text: qsTr("About")
+				iconName: "help-about"
+				onTriggered: pageStack.push(aboutPage)
```

Comment:
First pop all pages till rosterPage (`pageStack.pop(rosterPage);`) to prevent that the page is opened multiple times